template/src/main/twirl/ch.epfl.scala.index.views/artifact.scala.html

```diff
@@ -1,9 +1,9 @@
-@import ch.epfl.scala.index.model.Project
-@import ch.epfl.scala.index.model.Artifact
-@import ch.epfl.scala.index.model.Release
-@import ch.epfl.scala.index.model.misc.UserInfo
-@import ch.epfl.scala.index.model.release.{Dependency, JavaDependency, ScalaDependency, SemanticVersion}
-@import ch.epfl.scala.index.model.misc.GithubInfo
+@import ch.epfl.scala.index._
+@import model._
+@import misc.UserInfo
+@import release._
```

Comment:
`@import release._` should be `@import release.{Dependency, JavaDependency, ScalaDependency, SemanticVersion}`